src/SpannerClient/SpannerClientFactory.php

```diff
@@ -40,14 +40,13 @@ class SpannerClientFactory
     public function create()
     {
         $authCache = new SysVCacheItemPool();
-        $keyFileName = $_ENV[self::KEY_FILE_ENV_VARIABLE] ?? '';
+        $keyFileName = getenv(self::KEY_FILE_ENV_VARIABLE) ?? '';
```

Comment:
That's what we fixed to get environment variables either from Unix environment, or from .env[.local|.dist] file(s). So we need to use `$_ENV`.